Student/src/main/java/com/fnp/controller/StudentController.java

```diff
@@ -3,77 +3,63 @@
 import java.util.List;
 import java.util.Optional;
 
+import javax.validation.Valid;
+
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.context.annotation.Bean;
-import org.springframework.stereotype.Controller;
-import org.springframework.ui.Model;
-import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
 
-import com.fasterxml.jackson.core.JsonProcessingException;
-import com.fasterxml.jackson.databind.ObjectMapper;
-//import com.fnp.dao.StudentDao;
 import com.fnp.dto.Student;
+import com.fnp.dto.StudentModel;
 import com.fnp.service.StudentService;
 
-@Controller
+@RestController
 public class StudentController {
 
 	@Autowired
 	StudentService studentService;
-	
-	@PostMapping("/save")
-	public Model sendStudent(Student student, Model model) {
-		
-		Student s = studentService.saveStudent(student);
-		
-		model.addAttribute("studentId", s.getId());
-		model.addAttribute("studentName", s.getName());
-		model.addAttribute("studentEmail", s.getEmail());
-		model.addAttribute("studentCreator", s.getCreatedby());
-		model.addAttribute("studentUpdater", s.getUpdatedBy());
-		model.addAttribute("studentPhone", s.getPhone());
-		model.addAttribute("studentCreationDate", s.getCreatedOn());
-		model.addAttribute("studentUpdationDate", s.getLastUpdatedBy());
 
-		System.out.println(s);
-		System.out.println(s.getCreatedby());
-		return model;
+	@PostMapping(value = "/students", consumes = MediaType.APPLICATION_JSON_VALUE)
+	public ResponseEntity<Student> sendStudent(@Valid @RequestBody StudentModel studentModel,
+			BindingResult bindingResult) {
+
+		Student studentObj = null;
+		if (bindingResult.hasErrors()) {
+			return ResponseEntity.badRequest().body(studentObj);
```

Comment:
should not send the empty student,
if there are errors, then need to throw the exception